daemon/anonymous-statistics.sh.in

```diff
@@ -69,6 +69,7 @@ NETDATA_USE_PRIVATE_REGISTRY="${40}"
 NETDATA_CONFIG_OOM_SCORE="${41}"
 NETDATA_PREBUILT_DISTRO="${42}"
 
+[ ! -z "${43}" ] && NETDATA_FAIL_REASON=", \"netdata_fail_reason\": \"${43}\""
```

Comment:
@stelfrag See the warning here: `Use -n instead of ! -z.`

Reply:
Rewritten to always have the field